### PR TITLE
Greek translation

### PR DESCRIPTION
**Pull request by fractalbit**
Hello, i translated voyager to the Greek language (i am a native speaker). It is a complete translation, i only left some terminology intact that didn't make much sense to translate. I was also careful not to translate special directives like :string or \_START\_.

I absolutely **love** voyager and i hope you will accept my translation. Thank you!

### CODE REVIEW

**Comment by emptynick** [comment]
Thanks for your contribution!
Are you finished or do you want to push some more things?

**Comment by fractalbit** [comment]
Thank you for the quick reply!

I just spotted a small correction I need to make, but, since it is my first
ever PR I am not sure what to do.
Should I just "git push origin greek-translation" and the PR will be
updated?

Also, if i spot more mistakes in the future should i make a new PR after
this has been accepted?

Thank you for your time!

Στις Δευ, 13 Ιουλ 2020 στις 7:01 μ.μ., ο/η Christoph Schweppe <
notifications@github.com> έγραψε:

**Comment by emptynick** [comment]
You can just push commits like you did before and it will be updated here automatically.
Once it's merged you have to create a new PR.

Let me know when you are done

**Comment by fractalbit** [comment]
I think now it is ready. Thank you.

**Review by emptynick — APPROVED** [review]

File: publishable/lang/el/generic.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
+<?php
+
+return [
+    'is_rtl'                 => 'όχι',
```
Comment (MrCrayon) [inline publishable/lang/el/generic.php:4]: This field is used to check if language is rtl, so even if it's still going to work as far as is not literal `'true'` it's probably better if it's set as `'false'`.

**Review by MrCrayon — COMMENTED** [review]
If you can submit another PR with this fix would be great. 
Thanks!

File: publishable/lang/el/generic.php (hunk 1 above)
Comment (emptynick) [reply publishable/lang/el/generic.php:4]: Fixed [here](https://github.com/the-control-group/voyager/commit/aa44a8e40c72d4282e31f2df9637291c8dc069e1)

**Pull request merged**